### PR TITLE
Fix some bugs caused by sprinting for some characters and Add sprint bar for HUD

### PR DESCRIPTION
**Pull request by LeonardoTheMutant**

### CODE REVIEW

**Comment by luigi-budd** [comment]
jisk<<13
A bar is better than a percentage I say

**Comment by LeonardoTheMutant** [comment]
I think the same

File: src/Lua/main/healthncombat.lua [hunk 1]
```diff
@@ -229,6 +229,16 @@ addHook("PreThinkFrame", function()
 			else
 				player["srbz_info"].pressednext = false	
 			end
+
```
Comment (Jiskster) [inline src/Lua/main/healthncombat.lua:232]: please remove this

Comment (LeonardoTheMutant) [reply src/Lua/main/healthncombat.lua:232]: Please answer why

Comment (Jiskster) [reply src/Lua/main/healthncombat.lua:232]: this will collide with the new emote stuff i have

Comment (LeonardoTheMutant) [reply src/Lua/main/healthncombat.lua:232]: Then make it

**Pull request closed without merging**